PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -323,6 +356,14 @@ void PATElectronProducer::produce(edm::Event & iEvent, const edm::EventSetup & i
 	  const edm::RefToBase<reco::GsfElectron>& elecsRef = electrons->refAt(idx);
 	  Electron anElectron(elecsRef);
 	  anElectron.setPFCandidateRef( pfRef  );
+    if (addPuppiIsolation_) {
+      anElectron.setIsolationPUPPI((*PUPPIIsolation_charged_hadrons)[elePtr], (*PUPPIIsolation_neutral_hadrons)[elePtr], (*PUPPIIsolation_photons)[elePtr]);
+      anElectron.setIsolationPUPPINoLeptons((*PUPPINoLeptonsIsolation_charged_hadrons)[elePtr], (*PUPPINoLeptonsIsolation_neutral_hadrons)[elePtr], (*PUPPINoLeptonsIsolation_photons)[elePtr]);
+    }
+    else {
+      anElectron.setIsolationPUPPI(-999., -999.,-999.);
+      anElectron.setIsolationPUPPINoLeptons(-999., -999.,-999.);
+    }
```

Comment:
@ishvetso 
Lastly, would it be possible for you to keep the indentation as other lines?